mava/wrappers/jumanji.py

```diff
@@ -34,35 +34,82 @@
 
 
 class MultiAgentWrapper(Wrapper):
-    def __init__(self, env: Environment):
+    def __init__(self, env: Environment, has_global_state: bool = False):
         super().__init__(env)
         self._num_agents = self._env.num_agents
         self.time_limit = self._env.time_limit
+        self.has_global_state = has_global_state
 
     def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
         """Modify the timestep for `step` and `reset`."""
-        pass
+        return timestep
```

Comment:
I vote for defining it as abstract or maybe raising an error ( message: not implemented method).  What do you think?

``` suggestion
    @abstarctmethod
    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
        """Modify the timestep for `step` and `reset`."""
        ...

```

Reply:
Yes, I think this makes sense.